onnxruntime/core/providers/cpu/nn/conv_attributes.h

```diff
@@ -136,7 +136,9 @@ struct ConvAttributes {
                              " W: ", W->Shape().ToString().c_str());
     }
 
-    if (C != W->Shape()[1] * group) {
+    if (-1 == group) {
+      group = C / W->Shape()[1];
```

Comment:
we do not need to modify the value of `group`

Reply:
Moving out the logic.